Module.php

```diff
@@ -50,14 +52,15 @@ public function getViewHelperConfig()
 
     public function getServiceConfig()
     {
-        return array(
-            'aliases' => array(
+        return [
+            'aliases' => [
                 'lmcuser_laminas_db_adapter' => \Laminas\Db\Adapter\Adapter::class,
-            ),
-            'invokables' => array(
-                'lmcuser_register_form_hydrator' => \Laminas\Hydrator\ClassMethodsHydrator::class,
-            ),
-            'factories' => array(
+                'lmcuser_register_form_hydrator' => 'lmcuser_user_hydrator',
+                'lmcuser_base_hydrator' => ClassMethodsHydrator::class
```

Comment:
I guess `ClassMethodsHydrator::class` does not exist in servicemanager and should be retrieved from hydrators manager as it is hydrator not a service